From 1c00bca0f73d09bfd5d16b1c40f318bff78f3bdb Mon Sep 17 00:00:00 2001
From: Carlos Rafael Giani <dv@pseudoterminal.org>
Date: Fri, 23 Mar 2018 13:31:49 +0100
Subject: [PATCH] gst-libs: Remove library path switch from dependency_libs
 line

Yocto complains because it sees a -L/usr/lib path being used. Apparently,
GCC internally translates this path by appending the sysroot path, so
it isn't actually pointing towards the host's libraries. Still, remove
the -L switch for Yocto, since it isn't needed for the build.

Upstream-Status: Submitted [https://bugzilla.gnome.org/show_bug.cgi?id=794621]

Signed-off-by: Carlos Rafael Giani <dv@pseudoterminal.org>
---
 gst-libs/ext/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst-libs/ext/Makefile.am b/gst-libs/ext/Makefile.am
index 6cdc048..f45e71f 100644
--- a/gst-libs/ext/Makefile.am
+++ b/gst-libs/ext/Makefile.am
@@ -49,7 +49,7 @@ echo "  GEN      $1.la" && \
  echo "library_names=''" && \
  echo "old_library='$1.a'" && \
  echo "inherited_linker_flags=''" && \
- echo "dependency_libs=' -L$(libdir) $(if $2,$(foreach dep,$2,$(abs_builddir)/$(dep).la)) $(call find_library_la,$3 $(LIBM),$(LDFLAGS)) '" && \
+ echo "dependency_libs=' $(if $2,$(foreach dep,$2,$(abs_builddir)/$(dep).la)) $(call find_library_la,$3 $(LIBM),$(LDFLAGS)) '" && \
  echo "weak_library_names=''" &&  \
  echo "current=" && \
  echo "age=" && \
-- 
2.14.1

